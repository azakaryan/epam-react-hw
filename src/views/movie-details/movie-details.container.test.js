import React from 'react';
import { shallow } from 'enzyme';
import expectEnzymify from 'expect';
import enzymify from 'expect-enzyme';
import MovieDetails, { mapDispatchToProps } from './movie-details.container';

expectEnzymify.extend(enzymify());
const onLoad = () => {};

describe('MovieDetails Container Component', () => {
  it('should render correctly', () => {
    const component = shallow(<MovieDetails onLoad={onLoad} />);
    expect(component).toMatchSnapshot();
  });

  it('should not dispatch an action', () => {
    const dispatch = jest.fn();
    mapDispatchToProps(dispatch).onLoad();
    expect(dispatch.mock.calls[0][0]()).toBeTruthy;
  });
});