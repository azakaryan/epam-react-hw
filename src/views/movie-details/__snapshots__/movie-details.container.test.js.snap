// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieDetails Container Component should render correctly 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
